css/index.css

```diff
@@ -129,18 +225,162 @@ p {
 footer {
     width: 100%;
     background: black;
+    padding-bottom: 5%;
 }
 
 footer nav {
     width: 60%;
     display: flex;
     justify-content: space-between;
-    align-items: center;
+    align-items:center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: 1.4rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
-}
\ No newline at end of file
+}
+
+/*-----------------media queries----*/
```

Comment:
Little notes like this are invaluable. Knowing that media queries start here will stop you from continuing to scroll up.